src/main/java/uk/gov/ida/verifyserviceprovider/configuration/ConfigurationConstants.java

```diff
@@ -18,4 +18,28 @@
 
     String HUB_JERSEY_CLIENT_NAME = "VerifyHubMetadataClient";
     String MSA_JERSEY_CLIENT_NAME = "MsaMetadataClient";
+
+    interface EnvironmentUrls {
+
+        String PRODUCTION_HOST ="https://www.signin.service.gov.uk";
+        String INTEGRATION_HOST ="https://www.integration.signin.service.gov.uk";
+        String COMPLAINCE_HOST = "https://compliance-tool-reference.ida.digital.cabinet-office.gov.uk";
+
+
+        String PRODUCTION_SSO = PRODUCTION_HOST + "/SAML2/SSO";
+        String PRODUCTION_METADATA = PRODUCTION_HOST + "/SAML2/metadata/federation";
+
+        String PRODUCTION_TRUSTANCHOR_URI = PRODUCTION_HOST + "/SAML2/metadata/trust-anchor";
+        String PRODUCTION_METADATASOURCE_URI = PRODUCTION_HOST + "/SAML2/metadata/aggregator";
+
+        String INTEGRATION_TRUSTANCHOR_URI = INTEGRATION_HOST + "/SAML2/metadata/trust-anchor";
+        String INTEGRATION_METADATASOURCE_URI = INTEGRATION_HOST + "/SAML2/metadata/aggregator";
+        String INTEGRATION_SSO = INTEGRATION_HOST + "/SAML2/SSO";
+        String INTEGRATION_METADATA = INTEGRATION_HOST +  "/SAML2/metadata/federation";
+
+        String COMPLAINCE_SSO = COMPLAINCE_HOST + "/SAML2/SSO";
+        String COMPLAINCE_METADATA = COMPLAINCE_HOST +  "/SAML2/metadata/federation";
```

Comment:
Typo: COMPLAINCE -> COMPLIANCE

While we're looking at this class, please could you also put the constants for integration in the same order as the ones for production/compliance.

Obviously these are very minor issues and don't necessarily need to block a merge.

Reply:
Well spotted this one